pkg/apis/core/v1alpha1/defaults.go

```diff
@@ -119,6 +119,16 @@ func SetDefaults_Seed(obj *Seed) {
 	if obj.Spec.Settings.VerticalPodAutoscaler == nil {
 		obj.Spec.Settings.VerticalPodAutoscaler = &SeedSettingVerticalPodAutoscaler{Enabled: true}
 	}
+
+	if obj.Spec.Settings.DependencyWatchdog == nil {
+		obj.Spec.Settings.DependencyWatchdog = &SeedSettingDependencyWatchdog{}
+	}
+	if obj.Spec.Settings.DependencyWatchdog.Endpoint == nil {
+		obj.Spec.Settings.DependencyWatchdog.Endpoint = &SeedSettingDependencyWatchdogEndpoint{Enabled: true}
+	}
+	if obj.Spec.Settings.DependencyWatchdog.Probe == nil {
+		obj.Spec.Settings.DependencyWatchdog.Probe = &SeedSettingDependencyWatchdogProbe{Enabled: true}
+	}
```

Comment:
Nit: I think these should actually be in their own defaulting func, i.e. `SetDefaults_SeedSettingDependencyWatchdog`